AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_LightAdded.py

```diff
@@ -97,74 +182,299 @@ def AtomEditorComponents_Light_AddedToEntity():
+                        light_component.set_component_property_value(
+                            AtomComponentProperties.light('ESM exponent'), 5000)
+                        Report.result(Tests.edit_esm_exponent,
+                                      light_component.get_component_property_value(
+                                          AtomComponentProperties.light('ESM exponent')) == 5000)
+
+                # 20. Disable Shadows (re-enabled after test for game mode verification):
+                light_component.set_component_property_value(
+                    AtomComponentProperties.light('Enable shadow'), False)
+                general.idle_wait_frames(1)
+                Report.result(
+                    Tests.disable_shadows,
+                    light_component.get_component_property_value(
+                        AtomComponentProperties.light('Enable shadow')) is False)
+                light_component.set_component_property_value(
+                    AtomComponentProperties.light('Enable shadow'), True)
+
+            # Shutter tests for applicable light types:
+            if current_light_type in (2, 7):
+                # 21. Edit the Inner angle parameter:
+                light_component.set_component_property_value(
+                    AtomComponentProperties.light('Inner angle'), 0.5)
+                Report.result(Tests.edit_inner_angle,
+                              light_component.get_component_property_value(
+                                  AtomComponentProperties.light('Inner angle')) == 0.5)
+
+                # 22. Edit the Outer angle parameter:
+                light_component.set_component_property_value(
+                    AtomComponentProperties.light('Outer angle'), 90)
+                Report.result(Tests.edit_outer_angle,
+                              light_component.get_component_property_value(
+                                  AtomComponentProperties.light('Outer angle')) == 90)
+
+            # Disable/Enable Shutters tests:
+            if current_light_type == 2:
+                # 23. Disable Shutters:
+                light_component.set_component_property_value(
+                    AtomComponentProperties.light('Enable shutters'), False)
+                Report.result(
+                    Tests.disable_shutters,
+                    light_component.get_component_property_value(
+                        AtomComponentProperties.light('Enable shutters')) is False)
+
+                # 24. Enable Shutters:
+                light_component.set_component_property_value(
+                    AtomComponentProperties.light('Enable shutters'), True)
+                Report.result(
+                    Tests.enable_shutters,
+                    light_component.get_component_property_value(
+                        AtomComponentProperties.light('Enable shutters')) is True)
+
+            # Enable/Disable Both directions tests:
+            if current_light_type in (4, 5):
```

Comment:
same here use `LIGHT_TYPE` dict reference for the number

Reply:
Changed to:
if light_type in ('quad', 'polygon'):